torchbenchmark/util/backends/torchdynamo.py

```diff
@@ -21,14 +21,19 @@ def parse_torchdynamo_args(model: 'torchbenchmark.util.model.BenchmarkModel', dy
     return args, extra_args
 
 def apply_torchdynamo_args(model: 'torchbenchmark.util.model.BenchmarkModel', args: argparse.Namespace, precision: str):
-    torchdynamo.config.raise_on_backend_error = False
+    torchdynamo.config.raise_on_backend_error = True
```

Comment:
Not raise this error in CI, just for debug

Reply:
fixed